packages/integration-tests/src/test-projects/passwordless-auth/amplify/auth/resource.ts

```diff
@@ -4,6 +4,9 @@ export const auth = defineAuth({
   loginWith: {
     email: true,
   },
+  userAttributes: {
```

Comment:
I was getting the user is not verified error but because my account was not out of the sandbox mode. I need to verify that again, if not needed I'll remove it.